datasets/vggface2.py

```diff
@@ -23,9 +23,8 @@
 from torchvision import transforms
 
 import cv2
+import face_detection
 import kornia.geometry.transform as GT
```

Comment:
I think the copyright text should be updated for 2024.

Reply:
Realized in the last commit.